gridt/src/app/core/settings.service.ts

```diff
@@ -87,120 +94,35 @@ export class SettingsService {
    */
   private storeLocalSettings(settings: Settings): void {
     console.log(`Storing ${JSON.stringify(settings)} into the local storage.`);

```

Comment:
Printing out an object in the logs is not necessary. Maybe also debug is a more suitable level for this statement as well as some other logs you are making.

Reply:
I would actually recommend doing a search for 'log' and revisiting the necessity of all of them.